.env.example

```
@@ -235,6 +235,17 @@ EMAIL_REQUIRE_SENDER_AUTH_HEADERS=true
 MIGADU_API_USER=your_migadu_api_user
 MIGADU_API_KEY=your_migadu_api_key
 MIGADU_MAILBOX_DOMAIN=508.dev
+BREVO_API_KEY=
+BREVO_API_BASE_URL=https://api.brevo.com/v3
+BREVO_API_TIMEOUT_SECONDS=20.0
+BREVO_508_MEMBERS_NEWSLETTER_LIST_ID=
+BREVO_508_MEMBERS_NEWSLETTER_LIST_NAME=508 members
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BREVO_508_MEMBERS_NEWSLETTER_LIST_NAME key should go before the BREVO_API_BASE_URL key

(UnorderedKey)